packages/wix-ui-tpa/src/components/Dialog/Dialog.tsx

```diff
@@ -6,6 +6,7 @@ import { IconButton } from '../IconButton';
 import { ReactComponent as CloseIcon } from '../../assets/icons/Close.svg';
 import { TPAComponentProps } from '../../types';
 import { st, classes } from './Dialog.st.css';
+import ReactDOM from 'react-dom';
```

Comment:
Now we get on CI this:
`src/components/Dialog/Dialog.tsx(9,8): error TS1192: Module '"/home/builduser/agent00/work/42a4aad154f95b75/node_modules/@types/react-dom/index"' has no default export.`

Not sure why it's not exported as a module in the types. 😞

We can easily solve this by:
```suggestion
import * as ReactDOM from 'react-dom';
```

But in a second thought I think we should do the following since it's cleaner:
```suggestion
import { createPortal } from 'react-dom';
```

And obviously to change in the usage (line 140).